polyphemus/lib/etls/slack_notification_module.rb

```diff
@@ -0,0 +1,10 @@
+require_relative "../helpers"
+
+module Polyphemus::WithSlackNotification
+  include WithLogger
+
+  def slack(title, channel, msg)
```

Comment:
Interestingly, I implemented something similar, too, so I'll fold in my PR with yours after.